src/pfe/file-watcher/server/test/functional-test/suites/project/tests/project-action.ts

```diff
@@ -37,9 +37,9 @@ export function projectActionTest(socket: SocketIO, projData: ProjectCreation):
             "action": "disableautobuild",
             "returnKeys": ["statusCode", "status"],
             "statusCode": 200,
-            "socketEvent": projData.projectType.toLowerCase() === "docker" || process.env.IN_K8 ? [] : [eventConfigs.events.projectChanged],
-            "eventKeys": projData.projectType.toLowerCase() === "docker" || process.env.IN_K8 ? [] : [["operationId", "projectID", "ignoredPaths", "status", "host", "ports", "containerId", "logs"]],
-            "result": projData.projectType.toLowerCase() === "docker" || process.env.IN_K8 ? [] : [{
+            "socketEvent": process.env.IN_K8  || !project_configs.autoBuildEventCapabailities[projData.projectType] ? [] : [eventConfigs.events.projectChanged],
```

Comment:
For `autoBuildEventCapabailities` like I mentioned above, dont think we should work around it for tests but rather fix it in src code. Probably ask Rajiv's input regd it...

Reply:
Read comment https://github.com/eclipse/codewind/pull/387#discussion_r321511499